src/Compilers/Core/AnalyzerDriver/AnalyzerDriver.projitems

```diff
@@ -9,23 +9,11 @@
     <Import_RootNamespace>AnalyzerDriver</Import_RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="$(MSBuildThisFileDirectory)AnalyzerDriverHelper.cs">
```

Comment:
Ah, looks like you removed them from csprojs when you did the pass to apply the style conventions. Workspaces.csproj some for example has some that were removed.